src/js/components/Pagination/PaginationStep.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Box } from '../Box';
+import { Select } from '../Select';
+import { Text } from '../Text';
+
+export const PaginationStep = ({ step, onChange, options, ...rest }) => (
```

Comment:
Based on today's share-out, I wonder if we should support some default options. Can discuss what the reasonable defaults should be.

Reply:
Yes, maybe we can chat on these Im wondering if we should look at total then have defaults or just one set oof basic defaults I kinda helf off just because if we only have 40 items and then 25, 50, 100 might not make sense